pandas/core/config_init.py

```diff
@@ -539,7 +539,7 @@ def use_inf_as_na_cb(key) -> None:
     The default storage for StringDtype.
 """
 
-nullable_backend_doc = """
+dtype_backend_doc = """
 : string
     The nullable dtype implementation to return.
```

Comment:
I am ok with this too, but I'd like to make this more prominent here that this only works with use_nullable_dtypes as of right now

Reply:
I think it's also used with `convert_dtypes()` now and will (hopefully) be used in more places in the future. Instead of explicitly listing where this option is used, maybe we have some more generic statement about it still being experimental and only used in certain methods -- see the API docs for a specific method to see if it supports this option?

Reply:
Yep this sounds good to me. I am happy as long as we clarify that it is not generic yet.